src/main/resources/application.yaml

```diff
@@ -126,6 +126,11 @@ file-cleanup:
   cron: ${FILE_CLEANUP_CRON:0 15 * * * *}
   ttl: 12h
 
+old-letter-content-cleanup:
+  enabled: ${OLD_LETTER_CONTENT_CLEANUP_ENABLED:false}
```

Comment:
this job won't be running in AAT staging which is fine I think as you don't have any func tests relying on the job.

Otherwise `OLD_LETTER_CONTENT_CLEANUP_ENABLED` should be defined in values yaml with false value and then overrided wherever needed.

Reply:
Yup, I've made this config match the other AAT job (`DeleteOldFilesTask.java` that uses `FILE_CLEANUP_ENABLED` env var)